openrtb-core/pom.xml

```diff
@@ -3,10 +3,10 @@
 
   <artifactId>openrtb-core</artifactId>
   <packaging>jar</packaging>
-  <name>Google OpenRTB Core</name>
+  <name>IAB OpenRTB Core</name>
 
   <parent>
-    <groupId>com.google.openrtb</groupId>
+    <groupId>com.iabtechlab</groupId>
```

Comment:
I would expect this (and the other below) needs to be `com.iabtechlab.openrtb`. If not, why not?

Reply:
Good point!

Reply:
Fixed